Mathlib/CategoryTheory/Limits/Shapes/BinaryProducts.lean

```diff
@@ -306,11 +306,11 @@ theorem BinaryCofan.mk_inr {P : C} (ι₁ : X ⟶ P) (ι₂ : Y ⟶ P) : (Binary
 
 /-- Every `BinaryFan` is isomorphic to an application of `BinaryFan.mk`. -/
 def isoBinaryFanMk {X Y : C} (c : BinaryFan X Y) : c ≅ BinaryFan.mk c.fst c.snd :=
-    Cones.ext (Iso.refl _) fun j => by cases' j with l; cases l; repeat simp
+    Cones.ext (Iso.refl _) fun j => by obtain ⟨l⟩ := j; cases l; repeat simp
```

Comment:
Maybe
```suggestion
    Cones.ext (Iso.refl _) fun ⟨l⟩ => by cases l; repeat simp
```
unless this is a quotient pattern.

Reply:
Should have been replaced for the other category theory files touched.